packaging/bundle-validation/Dockerfile

```diff
@@ -47,5 +47,18 @@ RUN wget https://archive.apache.org/dist/spark/spark-$SPARK_VERSION/spark-$SPARK
     && tar -xf $WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz -C $WORKDIR/ \
     && rm $WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz
 ENV SPARK_HOME=$WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION
-COPY validate.sh .
 
+# TODO: the commands above can be moved to base images
```

Comment:
@jonvex let's also start using base images in this PR. the image building is taking >20min mostly just downloading. we need to have different image tags to use directly for different build profiles.